roofit/roofit/inc/RooPyBindFunction.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   Robin Syring, CERN 2024
+ *
+ * Copyright (c) 2023, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#ifndef ROOPYBINDFUNCTION_H
+#define ROOPYBINDFUNCTION_H
+
+#include "RooAbsReal.h"
+#include "RooListProxy.h"
+#include "RooArgList.h"
+
+class RooPyBindFunction : public RooAbsReal {
+public:
+   RooPyBindFunction(const char *name, const char *title, RooArgList &varlist);
+   RooPyBindFunction(const RooPyBindFunction &right, const char *name = nullptr);
+   virtual ~RooPyBindFunction();
```

Comment:
```suggestion
```
No need to implement a destructor if it doesn't do anything special (you see the implementation in the cxx file is empty). In this case, the compiler will automatically generate the default destructor for us. So you can delete the declaration here and the corresponding implementation in the cxx file.